components/status/StatusActionsMore.vue

```diff
@@ -42,10 +48,9 @@ function getPermalinkUrl(status: mastodon.v1.Status) {
   return null
 }
 
-async function copyLink(status: mastodon.v1.Status) {
-  const url = getPermalinkUrl(status)
```

Comment:
I think it would be better to override `getPermalinkUrl` instead. That method is also used inside the `shareLink` method (which is used on mobile and as it stands would still give the old incorrect link)

Reply:
Done!